recipes/geomstats/meta.yaml

```
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "geomstats" %}
+{% set version = "2.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ce7a31d3e8d449450132beeeae1559d159163529cc90d89e55fdeb80fae1844
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This looks like it's pure Python and could be noarch:
```suggestion
  number: 0
  noarch: python
```